kv/kvserver: TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic failed

kv/kvserver.TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5686761?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5686761?buildTab=artifacts#/) on master @ [cd1abbb3831ef8df9f792ff5dbb817f1fd36bc64](https://github.com/cockroachdb/cockroach/commits/cd1abbb3831ef8df9f792ff5dbb817f1fd36bc64):


```
=== RUN   TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic/initial_run
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvserver.TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5904178?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5904178?buildTab=artifacts#/) on master @ [2dc4b47ff79b6645ffe38c91ab6ee64c26b2963c](https://github.com/cockroachdb/cockroach/commits/2dc4b47ff79b6645ffe38c91ab6ee64c26b2963c):


```
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.bindUnaryServerInterceptorToHandler.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:795
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.NewServerEx.func3
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:263
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.bindUnaryServerInterceptorToHandler.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:795
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/auth.go:73
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.bindUnaryServerInterceptorToHandler.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:795
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.NewServerEx.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:232
        	            	  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:341
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.NewServerEx.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:230
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.bindUnaryServerInterceptorToHandler.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:795
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.makeInternalClientAdapter.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:734
        	            	  | github.com/cockroachdb/cockroach/pkg/util/tracing/grpcinterceptor.ClientInterceptor.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/tracing/grpcinterceptor/grpc_interceptor.go:225
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.getChainUnaryInvoker.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:879
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.makeInternalClientAdapter.func3
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:764
        	            	  | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
        	            	  | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:887
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:210
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:189
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2074
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1614
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1217
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:843
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:223
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:996
        	            	Wraps: (3) after 0.02s of attempting command
        	            	Wraps: (4) context deadline exceeded
        	            	Error types: (1) *roachpb.AmbiguousResultError (2) *errbase.opaqueWrapper (3) *errutil.withPrefix (4) context.deadlineExceededError
        	Test:       	TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic/after_restart
        	Messages:   	Expected success writing to liveness range
    --- FAIL: TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic/after_restart (1.12s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/kv/kvserver/client_raft_test.go
+++ b/pkg/kv/kvserver/client_raft_test.go
@@ -4787,7 +4787,7 @@ func TestDefaultConnectionDisruptionDoesNotInterfereWithSystemTraffic(t *testing
 
 		// Set a relatively short timeout so that this test doesn't take too long.
 		// We should always hit it.
-		withTimeout, cancel := context.WithTimeout(ctx, 20*time.Millisecond)
+		withTimeout, cancel := context.WithTimeout(ctx, 1*time.Second)
 		defer cancel()
 
 		// Write to the liveness range on the System class.

